hedera-node/hedera-app-spi/src/main/java/module-info.java

```diff
@@ -2,6 +2,7 @@
     requires transitive com.hedera.hashgraph.protobuf.java.api;
     requires static transitive com.github.spotbugs.annotations;
     requires com.swirlds.common;
+    requires com.google.protobuf;
```

Comment:
DO we really want to have this dependency at this point?

Reply:
Removed the dependency. Thanks !